newIDE/app/src/EventsFunctionsList/index.js

```diff
@@ -123,6 +133,54 @@ export default class EventsFunctionsList extends React.Component<Props, State> {
     );
   };
 
+  _getFunctionThumbnail = (eventsFunction: ?gdEventsFunction) => {
+    switch (eventsFunction.getFunctionType()) {
+      default:
+        return 'res/functions/function32.png';
+      case 0:
+        if (
```

Comment:
I wonder if it would be a godd idea to have a nested switch:
```js
switch (eventsFunction.getName()) {
  case 'onSceneLoaded':
  case 'onFirstSceneLoaded':
  case 'onCreated':
    return 'res/functions/create32.png';
  case ...
}
```
You could move up the 3 other cases that are shorter, and have this longer case as the last one for clarity's sake.